keyboards/herevoland/buff75/halconf.h

```diff
@@ -0,0 +1,20 @@
+/* * Copyright 2024 楽（HereVoLand @Vem）
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#define HAL_USE_PWM TRUE
```

Comment:
Configuring PWM and then not using it for anything is incorrect.

If the intention is to configure the PWM WS2812 driver, then additional config is required.